### PR TITLE
Add the new voting rules of the SIP Process

### PR DESCRIPTION
**Pull request by jvican**
These are the updated voting rules. I have cherry-picked some examples to show
how they work. The examples are illustrated for a Committee of 9 members
instead of 8, which is our current size. I've chosen an odd number to
illustrate the voting rules for Committee sizes whose 50% is not a natural number.

Suggestions are very much welcome!

@SethTisue, you may be interested in having a look.

Review by @odersky | @SethTisue | @heathermiller.

### CODE REVIEW

**Comment by jvican** [comment]
Thanks to @dwijnand for helping me out with the wording 👍

File: sips/sip-submission.md [hunk 1]
```diff
@@ -220,22 +219,61 @@ becomes the reviewer. The main tasks of the reviewer are the following:
 3. Provide them feedback from the discussions in the meetings, and
 4. Explain the latest progress in every meeting.
 
+### SIP meetings
+
+SIP meetings are scheduled montly by the Process Lead, and require a quorum of
```
Comment (dwijnand) [inline sips/sip-submission.md]: derp, s/montly/monthly/

Comment (dwijnand) [inline sips/sip-submission.md]: My typo.

Comment (jvican) [inline sips/sip-submission.md]: Will fix now.

**Comment by SethTisue** [comment]
LGTM

**Comment by heathermiller** [comment]
Thanks!

**Pull request merged**